### PR TITLE
Add more detail to summary in cloudtrail alert

### PR DESCRIPTION
**Pull request by ajvb**
* Adds the identity that performed the action (usually a username)
* Adds support for a "resource selector" that will include what resource
the action was performed on.

### CODE REVIEW

File: moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua [hunk 1]
```diff
@@ -127,7 +147,15 @@ function process_message()
 
         if match_counter == #event.fields then
             local id = string.format("%s - %s", event.description, event_id)
-            local s = string.format("%s in %s", event.description, get_account_name(account_id))
+            local s = string.format("%s in %s by %s", event.description, get_account_name(account_id), get_identity_name() or "unknown")
+            if event.resource then
+              s = string.format(
+                  "%s on %s",
+                  s,
+                  read_message(string.format("Fields[%s]", event.resource))
```
[the remark this answers is not in the document]
Comment (ajvb) [reply moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua]: Oh, yes absolutely.

File: moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua [hunk 2]
```diff
@@ -101,6 +104,23 @@ function get_account_name(account_id)
     return account_id
 end
 
+function get_identity_name()
+  local identity_type = read_message("Fields[userIdentity.type]")
+
+  local key = ""
```
Comment (trink) [inline moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua]: unused

File: moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua [hunk 3]
```diff
@@ -101,6 +104,22 @@ function get_account_name(account_id)
     return account_id
 end
 
+function get_identity_name()
```
Comment (ameihm0912) [inline moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua]: I think the indenting might be off in here?

Comment (ajvb) [reply moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua]: yeah it is, fixed.

**Pull request merged**